code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -39,7 +39,11 @@
 /// Don't spam the "HEY I'M COMING" messages
 	var/last_newpatient_speak = 0
 /// How much healing do we do at a time?
-	var/heal_amount = 2.5
+	var/heal_amount = 1
+/// How long do we apply the surgery bonus trait?
+	var/surgery_bonus_trait_duration = (15 SECONDS)

```

Comment:
white kits are more common, they aren't competitive.

Blu/Gre kits do nothing special unless the bot does not apply the trait, in which case it would function like old.